CONTRIBUTING.md

```diff
@@ -128,7 +128,25 @@ To run the installation script:
 **Important Notes:**
 - It is recommended to always use `latest` as the value for the `wp-version` argument.
 - The installation takes place into a temporary directory, which means that the related files will get deleted in a day or so. The way to remediate this is just to re-run the installation script whenever needed.
-- The database will be completely overwritten every time integration tests are executed. 
+- The database will be completely overwritten every time integration tests are executed.
+
+#### Installing pcov
+
+To be able to run coverage commands, you'll need to install `pcov`:
+
+```
+pecl install pcov
```

Comment:
Should we add it as dev dependency in composer?

Reply:
I don't think this is possible, as `pcov` is a [PECL extension](https://www.php.net/manual/en/install.pecl.php).